draft-ietf-jsonpath-base.md

```diff
@@ -1474,13 +1474,23 @@ Queries:
 
 ## Normalized Paths
 
-A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
-the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
-`$['book'][3]` and `$['book'][5]`. For a given JSON value, there is a one to one correspondence between the value's
-nodes and the Normalized Paths that identify these nodes.
-Note that there is precisely one Normalized Path that identifies each node.
+A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node,
+for example, `$['book'][3]`.
+
+A Normalized Path provides a concrete string representation of a node in a nodelist.
+Consequently, a JSONPath implementation may output Normalized Paths instead of, or in addition to, the values identified by these paths.
+For example, the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
+`$['book'][3]` and `$['book'][5]`.
+
+Normalized Paths are used in this document to clarify the contents of nodelists in examples.
```

Comment:
perhaps "show" instead of "clarify" ... ?

Reply:
> perhaps "show the resulting" instead of "clarify the contents of" ... ?

Yes. Adopted in 761f2608cc30ef5c1a3a59c9f091144b09e69ac8.

Reply:
Also consider "represent"

Reply:
I considered "represent", but "show" is more concrete, and is perhaps a brief respite for the reader from one-to-one correspondences and other abstract language. :-)